### ISSUE TITLE
Add "skip to main content" link

### ISSUE THREAD

**Issue by cjg89** [UCF/Pegasus-Theme#112]
Add a "skip to main content" link that becomes visible on focus and is the first tabbable element on any page/story.  The link should be an anchor pointing to `<main>`, allowing keyboard users to skip the site's header navigation.

Example: https://github.com/UCF/UCF-WordPress-Theme/blob/master/header.php#L7-L8
